trade-finance-manager-ui/server/api.js

```diff
@@ -112,22 +110,49 @@ const updateParty = async (id, partyUpdate) => {
   return response;
 };
 
-const updateFacility = async (id, facilityUpdate) => {
-  const updateVariables = {
-    id,
-    facilityUpdate,
-  };
-  const response = await apollo('PUT', updateFacilityMutation, updateVariables);
-  return response;
+const updateFacility = async (id, facilityUpdate, token) => {
+  try {
+    const isValidFacilityId = isValidMongoId(id);
+
+    if (!isValidFacilityId) {
+      console.error('updateFacility: Invalid facility id provided: %s', id);
+      return { status: 400, data: 'Invalid facility id' };
+    }
+
+    const response = await axios({
+      method: 'put',
+      url: `${TFM_API_URL}/v1/facilities/${id}`,
+      headers: generateHeaders(token),
+      data: facilityUpdate,
+    });
+
+    return response.data;
+  } catch (error) {
+    console.error('Unable to update facility request %O', error);
```

Comment:
I know that some of the error messages have the word request in but can we delete it I think it's misleading